extra-libs/libinput/autobuild/defines

```diff
@@ -1,6 +1,8 @@
 PKGNAME=libinput
 PKGSEC=libs
-PKGDEP="mtdev systemd libevdev libwacom gtk-3"
+PKGDEP="mtdev systemd libevdev libwacom gtk-3 \
```

Comment:
Is `gtk-3` still necessary? Also, if `gtk-4` is necessary for one of the programs, use DEP.